UI/window-basic-properties.cpp

```diff
@@ -114,15 +114,20 @@ OBSBasicProperties::OBSBasicProperties(QWidget *parent, OBSSource source_)
 		obs_display_add_draw_callback(preview->GetDisplay(),
 				OBSBasicProperties::DrawPreview, this);
 	};
-
+	
 	enum obs_source_type type = obs_source_get_type(source);
 	uint32_t caps = obs_source_get_output_flags(source);
 	bool drawable_type = type == OBS_SOURCE_TYPE_INPUT ||
 		type == OBS_SOURCE_TYPE_SCENE;
-
-	if (drawable_type && (caps & OBS_SOURCE_VIDEO) != 0)
+	bool drawablePreview = (caps & OBS_SOURCE_VIDEO) != 0;
```

Comment:
Super-nitpick: call it `drawable_preview` instead of `drawablePreview`, to fit the local naming convention (should probably also rename it in the other file if you make it local)